repository-data/application/src/main/resources/hcm-config/namespaces/hee/expanderPanel.yaml

```diff
@@ -69,4 +76,17 @@ definitions:
             wicket.id: ${cluster.id}.field
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-              ckeditor.config.appended.json: '{ removeButtons: "PickImage", forcePasteAsPlainText: true }'
+              ckeditor.config.appended.json: '{ removeButtons: "PickImage,Table",
+                forcePasteAsPlainText: true }'
+          /contentBlocks:
+            jcr:primaryType: frontend:plugin
+            caption: Expander content objects
+            compoundList: hee:ActionLink,hippostd:html,hippogallerypicker:imagelink,hee:insetReference,hee:mediaEmbedReference,hee:tableReference,hee:richTextReference
```

Comment:
Could we remove `hee:tableReference` block as it doesn't required to be supported per ticket/spec ?